### DISCUSSION TITLE
Not showing leaderboard score

### DISCUSSION THREAD

**Discussion by vijaychandra1910** [recodehive/awesome-github-profiles#639] [Q&A] [answered]
My score is not showing to the leaderboard. What to do. ? 
![Screenshot 2024-10-06 164242](https://github.com/user-attachments/assets/2d24ecfc-22c0-4d5a-aaa2-5c3dc09f9a72)
My pull request is also merged.

**Comment by vijaychandra1910** [comment, marked as answer]
@sanjay-kv  please give suggestion.

**Reply by sanjay-kv** [reply]
its take 12 hr of time to show on leaderboard. leaderboard are not instant
